lib/kitchen/directions/bake_notes/bake_note_subtitle.rb

```diff
@@ -3,11 +3,15 @@
 module Kitchen
   module Directions
     module BakeNoteSubtitle
-      def self.v1(note:)
+      def self.v1(book:, note:)
         title = note.title&.cut
 
         return unless title
 
+        # Store label information
+        note_label = title.children
+        book.document.pantry(name: :link_text).store note_label, label: note.id
```

Comment:
Can be `note.pantry` which means you can get rid of the addition of `book` into this class.